Add `Getting Started` page (#374)
ResearchComputing/Documentation: docs/getting_started/logging-in.md

# Logging In

```{tip}

> Do you need help navigating our documentation? For new and experienced users, we recommend reviewing our [Navigating CURC Documentation](./navigating_docs.md) page, which includes flowcharts for core areas of our documentation that all users should be aware of. 

Reviewer: ```suggestion
Need help navigating our documentation? Whether you're new or experienced, we recommend reviewing our [Navigating CURC Documentation](./navigating_docs.md) page. It includes flowcharts covering the core areas all users should be aware of.
```

```

## Getting a CURC account

> All individuals who would like to access CU Research Computing (CURC) HPC systems (Alpine and Blanca), Open OnDemand, and storage solutions must have an associated CURC account. CURC systems are utilized by a variety of institutions including CU Boulder, Anschutz Medical Campus (AMC), Colorado State University (CSU), and institutions who are members of the Rocky Mountain Advanced Computing Consortium (RMACC). Given CURC systems host a variety of institutions, each institution has its own method for creating a CURC account. To simplify this process, below we provide tabs that specify instructions on getting a CURC account based on the institution you are affiliated with. 

Reviewer: ```suggestion
All individuals who would like to access CU Research Computing (CURC) HPC systems (Alpine and Blanca), Open OnDemand, and storage solutions must have an associated CURC account. CURC systems are utilized by a variety of institutions including CU Boulder, Anschutz Medical Campus (AMC), Colorado State University (CSU), and institutions that are members of the Rocky Mountain Advanced Computing Consortium (RMACC). Given CURC systems host a variety of institutions, each institution has its own method for creating a CURC account. To simplify this process, below we provide tabs that specify instructions on getting a CURC account based on the institution you are affiliated with. 
```

The process for obtaining a CURC account for CU Boulder users is straightforward. Simply navigate to our [Account Request](https://rcamp.rc.colorado.edu/accounts/account-request/create/organization) page, select **"University of Colorado Boulder"**, fill out the provided information, and select **"Verify & Continue"**. Once the form has been completed and submitted, an automatic process will provision your account. Once you've received a CURC account, you will need to enroll in CU Boulder's Duo two-factor authentication service. After installing the Duo mobile app to your phone (available via Apple App Store or Google Play Store), visit [https://duo.colorado.edu](https://duo.colorado.edu) to enroll. 

````{important}
- Research Computing accounts are separate from your CU accounts. You must sign up for an individual Research Computing account if you wish to access Research Computing resources.
- Once your account is provisioned, please wait 15 minutes before signing in
- If you are having issues with DUO, please consult our [Duo Frequently Asked Questions](./faq.md#duo-multi-factor-authentication) section
````

`````

`````{tab-item} CSU
:sync: logging-in-csu

> The process for obtaining a CURC account for CSU users is straightforward. Simply navigate to our [Account Request](https://rcamp.rc.colorado.edu/accounts/account-request/create/organization) page, select **"Colorado State University"**, fill out the provided information, and select **"Verify & Continue"**. Once the form has been completed and submitted, an automatic process will provision your account. Once you've received a CURC account, you will need to enroll in Duo two-factor authentication service. For more information on setting up Duo, please visit CSU's [Duo Two Factor Authentication](https://it.colostate.edu/duo-two-factor-authentication/) page. For additional information on using CURC resources as a CSU user, please consult [CSU's Getting Started with Alpine](https://it.colostate.edu/research-computing-and-cyberinfrastructure/compute/get-started-with-alpine/) documentation.  

Reviewer: Updated the link - the old one went to a 404 Page not found.

```{important}
- Once your account is provisioned, please wait 15 minutes before signing in. 
```

`````

`````{tab-item} AMC
:sync: logging-in-amc

> Currently, AMC users must go through various steps to create a CURC account. These steps are outlined below: 

Reviewer: ```suggestion
Currently, AMC users must go through the following steps to create a CURC account: 
```

```

`````
``````

## Getting access to CURC resources 

Once you have obtained a CURC account, you can proceed to logging in to CURC resources. Users accessing CURC resources will be connected to a login node. A login node is an outward-facing node within the research computing environment that users can connect to from their local machines. Once on a login node, users can perform a limited number of tasks:

+ Edit files
+ Transfer data
+ Submit jobs
+ Access storage resources  

```{important}  
The [login node policy](../additional-resources/policies.md#login-nodes) states that login nodes should not be used for resource-intensive tasks such as running code. For all other tasks, users should run batch jobs, interactive jobs, or use the compile nodes. 
```

Similar to obtaining an account, the process of logging in to CURC resources can vary based on the institution you are affiliated with. In the tab sections below we provide details on how each institution can gain access to our login nodes.

(tabset-ref-get-access)=
`````{tab-set}
:sync-group: tabset-logging-in

````{tab-item} CU Boulder
:sync: logging-in-ucb

> For CU Boulder users, the following methods are available for logging in to CURC resources:

Reviewer: ```suggestion
For CU Boulder users, the following methods are available for logging into CURC resources:
```

- Utilizing Secure Shell (SSH) from your computer's terminal to `ssh` into a login node
    - For more information, on how to do this, see [Terminal access for different operating systems](#terminal-access-for-different-operating-systems) below
- Using Open OnDemand to obtain access to a terminal or launch interactive jobs 
    - See our [Open OnDemand (Browser Based HPC Portal)](../open_ondemand/index.md) documentation for more information
````

````{tab-item} CSU
:sync: logging-in-csu

For CSU users, the following methods are available for logging in to CURC resources:
- Utilizing Secure Shell (SSH) from your computer's terminal to `ssh` into a login node
    - For more information, on how to do this, see [Terminal access for different operating systems](#terminal-access-for-different-operating-systems) below

>     - For CSU specific guides for this process, please refer to the **"Remote Login"** section of CSU's [Get Started with Alpine](https://it.colostate.edu/research-computing-and-cyberinfrastructure/compute/get-started-with-alpine/#) page

Reviewer: ```suggestion
    - For CSU specific guides for this process, please refer to the **"Remote Login"** section of CSU's [Get Started with Alpine](https://www.research.colostate.edu/rit/research-computing-cyberinfrastructure/get-started-with-alpine/) page
```
